static/js/containers/ProfileImage.js

```diff
@@ -81,29 +80,20 @@ class ProfileImage extends React.Component<ProfileImageProps> {
   render() {
     const { profile, imageSize, editable, onClick } = this.props
 
-    const imageUrl = makeProfileImageUrl(
-      profile,
-      imageSize === "micro" || imageSize === "small"
-    )
+    const imageUrl = profile
+      ? imageSize === "micro" || imageSize === "small"
+        ? profile.profile_image_small
+        : profile.profile_image_medium
+      : defaultProfileImageUrl
```

Comment:
Logic here is harder to follow than necessary, IMO. This double ternary and the ternary below for the profile name would be more readable like this:

```javascript
let imageUrl, profileName
if (profile) {
  imageUrl = imageSize === "micro" || imageSize === "small" ? profile.profile_image_small : profile.profile_image_medium
  profileName = profile.name
} else {
  // ...
}
```